documentation/UserGuide.md

```diff
@@ -159,4 +159,14 @@ After the first launch, the app uses the bsaemap layers that it loaded previousl
 
 ### Simulating messages
 
-You can run the Message Simulator application included in the [Vehicle Commander](https://github.com/Esri/vehicle-commander) template ([source](https://github.com/Esri/vehicle-commander/tree/master/source/MessageSimulator), [sample messages](https://github.com/Esri/vehicle-commander/blob/master/source/MessageSimulator/SampleMessages/AFM_Sample_MSGS-719.xml). This is especially useful for testing and demonstration purposes. Note that these simulated messages will not make it to Squad Leader running on an emulator ([more info](#running-on-an-android-emulator)).
+You can run the [GeoMessage Simulator application](https://github.com/Esri/geomessage-simulator-qt) to send messages to Squad Leader. GeoMessage Simulator is especially useful for testing and demonstration purposes. Note that these simulated messages will not make it to Squad Leader running on an emulator ([more info](#running-on-an-android-emulator)). (TODO add link to compiled GeoMessage Simulator when it is posted)
+
+Squad Leader supports the "removeall" Geomessage action to remove all messages of a certain type (e.g. position_report, chemlight, spot_report). Here's an example that removes all position reports:
+
+    <geomessages>
+        <geomessage v="1.0">
```

Comment:
Probably. Looking in the code for Vehicle Commander and Squad Leader, I can only find one meaningful usage of the GeoMessage version. In https://github.com/Esri/vehicle-commander-java/blob/master/source/VehicleCommander/src/com/esri/vehiclecommander/controller/VehicleStatusController.java , if the version is 1.0, the outgoing message type is "sys_msg". Otherwise, the outgoing message type is "system_msgs" (and I'm not sure whether it should be "system_msgs" or "system_msg"). Other than that, we do nothing. So yes, we need to document it somewhere, but first we probably need to figure out what 1.0 and 1.1 actually mean. Let's create an issue in https://github.com/Esri/geomessage-simulator-qt .